lib/order.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'takeaway'
+class Order < Menu
+  
```

Comment:
Oh very nice use of inherited classes to put the menu functionality into the Order class.